test/integration/other/database-indexes.js

```diff
@@ -0,0 +1,73 @@
+const { sql } = require('slonik');
+const { curry, equals, slice } = require('ramda');
+
+const { testContainer } = require('../setup');
+
+const startsWith = curry((prefix, arr) => equals(prefix, slice(0, prefix.length, arr)));
+
+const idxNameFor = fk => `idx_fk_${fk.local_tbl_name}_${fk.local_col_names}`;
+const colsForIdx = fk => fk.local_col_names.map(col => `"${col}"`).join(', ');
+const createIdxStatement = fk => `CREATE INDEX ${idxNameFor(fk)} ON "${fk.local_tbl_name}" (${colsForIdx(fk)});`;
```

Comment:
```suggestion
const createIdxStatement = fk => `CREATE UNIQUE? INDEX ${idxNameFor(fk)} ON "${fk.local_tbl_name}" (${colsForIdx(fk)});`;
```

@brontolosone I'm thinking of changing to this.  This would mean that the generated statements can't be blindly copy/pasted into a migration (¬;